relevanceai/dataset_api/dataset.py

```diff
@@ -612,6 +612,11 @@ def head(
                 )
                 return pd.json_normalize(head_documents).head(n=n)
 
+    def _repr_html_(self):
+        return pd.json_normalize(
+            self.get_documents(dataset_id=self.dataset_id)
+        )._repr_html_()
```

Comment:
can this be `show_json` ? We want to ideally add support for images/audio if possible!

Reply:
Do you mean to make it like a try/except like in `head`?

Reply:
Yep - or can you just use `head` ?

Reply:
Reusing `head` would not work, I resolved it in a different way.